gravitee-apim-portal-webui-next/src/app/api-details/subscribe-to-api/subscribe-to-api-checkout/subscribe-to-api-checkout.component.html

```diff
@@ -0,0 +1,64 @@
+    
+            http://www.apache.org/licenses/LICENSE-2.0
+    
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<mat-card-header class="subscriptions-header">
+  <div class="m3-title-large">Checkout</div>
+  <div class="subscriptions-header__content">This plan does not need any subscription to consume the API.</div>
+</mat-card-header>
+
+@if (subscriptionDetails | async; as details) {
+  <div class="subscriptions-details__container">
+    <mat-card appearance="outlined" class="subscriptions-details__card">
+      <mat-card-header>
+        <div class="m3-title-medium subscriptions-details__header" i18n="@@subscriptionDetailsHeader">Subscription</div>
+      </mat-card-header>
+      <mat-card-content>
+        <div class="subscriptions-details__content-row subscriptions-details__content-info">
+          <p class="m3-title-small subscriptions-details__content-row-title" i18n="@@subscriptionDetailsHeaderPlan">Plan:</p>
+          <p class="m3-body-medium">{{ subscribeForm.value.plan.id }}</p>
+        </div>
+        @for (plans of subscribeForm.value.plan.plan; track plans) {
+          <div class="subscriptions-details__content-row subscriptions-details__content-info">
+            <p class="m3-body-medium">{{ plans }}</p>
+          </div>
+        }
+      </mat-card-content>
+    </mat-card>
+    <mat-card appearance="outlined" class="subscriptions-details__card">
+      <mat-card-header>
+        <div class="m3-title-medium subscriptions-details__header" i18n="@@subscriptionDetailsApiAccessHeader">API access</div>
+      </mat-card-header>
+      <mat-card-content>
+        <mat-form-field appearance="outline" class="subscriptions-details__field">
+          <mat-label i18n="@@subscriptionDetailsApiKeyBaseUrl">Base URL</mat-label>
+          <input matInput [value]="details.entrypoints[0]" readonly />
+          <button mat-icon-button matSuffix [cdkCopyToClipboard]="details.entrypoints[0]">
+            <mat-icon>content_copy</mat-icon>
+          </button>
+        </mat-form-field>
+        <mat-form-field appearance="outline" class="subscriptions-details__field">
+          <mat-label i18n="@@subscriptionDetailsApiKeyCommandLine">Command line</mat-label>
+          <input matInput [value]="formatCurlCommandLine(details.entrypoints[0])" readonly />
+          <button mat-icon-button matSuffix [cdkCopyToClipboard]="formatCurlCommandLine(details.entrypoints[0])">
+            <mat-icon>content_copy</mat-icon>
+          </button>
+        </mat-form-field>
+      </mat-card-content>
+    </mat-card>
+  </div>
+}
+<button (click)="onChangeStepper()" class="subscribe-checkout__button-back" mat-flat-button color="primary">Back</button>
```

Comment:
```suggestion
<button (click)="goBackStepper()" class="subscribe-checkout__button-back" mat-flat-button color="primary">Back</button>
```
or something like that 🤔 ?

Reply:
Done